test/UnitCopyPaste.cpp

```diff
@@ -374,6 +374,15 @@ class UnitCopyPaste : public UnitWSD
         if (!fetchClipboardAssert(_clipURI, "text/plain;charset=utf-8", "herring"))
             return;
 
+        // Test setting HTML clipboard:
+        std::string html("<!DOCTYPE html><html><body>myword</body></html>");
+        if (!setClipboard(_clipURI, buildClipboardText(html, "text/html"), HTTPResponse::HTTP_OK))
```

Comment:
Ah, if I use buildClipboardText(), then the payload will be in the mimetype-size-bytes format, the test should avoid that to actually cover the new functionality.